Dockerfile

```
@@ -495,6 +495,64 @@ RUN \
 
 # =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
 
+FROM sdk-go as sdk-govc
+
+USER root
+RUN \
+  mkdir -p /usr/share/licenses/govmomi && \
+  chown -R builder:builder /usr/share/licenses/govmomi
+
+ARG ARCH
+ARG TARGET="${ARCH}-bottlerocket-linux-gnu"
+ARG GOVMOMIVER="0.26.0"
+ARG GOVMOMISHORTCOMMIT="34586b6"
+ARG GOROOT="/home/builder/go"
+ARG GOPATH="${GOROOT}/go"
```

I'm not a fan of pushing non-stdlib sources into `GOROOT` this way. This implicitly supplies the package globally across all compilations which use the same `GOROOT`, rather than the usual `GOPATH` namespacing.

Can we instead use something like:

 ```suggestion
ARG GOPATH="${GOROOT}/gopath"
```

---

I wasn't crazy about it either, but (unless I misread) that is what we have going on in `sdk-go`. I'd like to stay consistent, so I think we should change it in `sdk-go` as well.

---

The `sdk-go` case is writing out the Go installation, right? Or are you seeing something else? In the case of the installation root, that's where Go's stdlib sources & intermediates should be installed.